app/scripts/glossaire.sh

```diff
@@ -249,6 +257,14 @@ function updateStandardRepo() {
             then
                 echored "Cache doesn't exist for ${repo_name}/${locale_code}"
                 updated_locale=1
+            else
+                php -l $cache_file 2>&1 1>/dev/null
```

Comment:
Is this supposed to echo the parse error in the console? If it’s expected, everything looks good